launch/package.xml

```diff
@@ -25,6 +25,7 @@
   <exec_depend>python3-importlib-metadata</exec_depend>
   <exec_depend>python3-lark-parser</exec_depend>
   <exec_depend>python3-yaml</exec_depend>
+  <exec_depend>python3-typing-extensions</exec_depend>
```

Comment:
One thing to note is that, for example, Debian has `python3-typing-extensions` version 4.4.0 (https://github.com/ros2/rclpy/pull/1458). It would be worth quickly checking that everything we import from `typing_extensions` here is available in <=4.4.0.

Reply:
Double checking `python3-typing-extensions` 4.4.0 does have both the `Self` type as well as `TypeAlias`.

Reply:
Thanks for checking.